### DISCUSSION TITLE
Emojis cut off when using polls

### DISCUSSION THREAD

**Discussion by papertek** [papertek/CyDJ#28] [Q&A] [answered]
_Originally posted by @eljulidi1337 in https://github.com/papertek/CyDJ/discussions/10#discussioncomment-1884689_

**Comment by eljulidi1337** [comment]
https://cdn.discordapp.com/attachments/560461680979476480/925850958729195530/unknown.png
The emote on the BR is always the same for me and it's kinda cut off... I don't know how it got this way but if I refresh it's still there. I refreshed once and it was gone. I refreshed again and it reappeared. I just got a new one and that one is also cut off: https://cdn.discordapp.com/attachments/560461680979476480/926189348288884736/unknown.png
I am on PC and using Opera GX.

**Reply by papertek** [reply]
seems like an opera gx problem, try another browser like chrome to see if anything changes

**Comment by papertek** [comment, marked as answer]
user confirmed it's an opera gx problem, not on our end.
